pipelines/rj_crm__wetalkie_api_hsm_info/tasks.py

```
@@ -0,0 +1,173 @@
+from typing import List, Dict, Any
+from datetime import datetime
+
+from prefect import task
+from iplanrio.pipelines_utils.logging import log
+from iplanrio.pipelines_utils.env import getenv_or_action
+
+from pipelines.rj_crm__wetalkie_api_hsm_info.constants import Constants
+
+
+@task
+def get_wetalkie_token(infisical_secret_path: str = None) -> str:
```

Ruff (0.15.0)

[warning] Unused function argument: `infisical_secret_path`

(ARG001)